src/Symfony/Component/Console/Helper/Helper.php

```diff
@@ -72,6 +72,18 @@ public static function substr($string, $from, $length = null)
         return mb_substr($string, $from, $length, $encoding);
     }
 
+    /**
+     * Returns the text within a portion of a string, using mb_substr if it is available.
```

Comment:
We could remove `... using mb_substr if it is available` from the description because it's an internal detail and in the PHPdoc we usually only explain what we do but not how we do that.

Also, the description is a bit confusing `Returns the text within a portion of a string ...` because the function name means `Replace a part of the given string...`

Reply:
I had put the `mb_substr` detail to be consistent with the others methods of this helper, but effectively, that isn't really useful, so it's deleted now.

About the description, I reworded to `Replaces text within a portion of a string` (the same description than in the [PHP doc](http://php.net/manual/en/function.substr-replace.php))